src/content/docs/apm/agents/nodejs-agent/installation-configuration/es-modules.mdx

```diff
@@ -78,6 +78,13 @@ You can add the loader to your application by using the Node.js `--experimental-
 node --experimental-loader newrelic/esm-loader.mjs your-program.js
 ```
 
+## Custom instrumentation [#custom-instrumentation]
```

Comment:
I think you're going to want to update this [page](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api/#custom-instrumentation) and perhaps cross link the ESM section from here